src/path_rewriting.rs

```diff
@@ -264,34 +291,36 @@ pub fn rewrite_paths(
         };
 
         // Get absolute path to the source file.
-        let (abs_path, rel_path) = get_abs_path(&source_dir, rel_path, &mut cache);
-
-        if to_ignore_globset.is_match(&rel_path) {
-            return None;
-        }
+        if let Some((abs_path, rel_path)) = get_abs_path(&source_dir, rel_path, &mut cache) {
```

Comment:
Nit: You could check if None here and return early, so you don't have to incraese the indentation for all the rest.